### PR TITLE
fix: annotation zoom changes on different aspect

### PR DESCRIPTION
**Pull request by WoodNeck**
## Details
This guarantees the same model size on different aspects of the canvas element

### CODE REVIEW

**Review by qpitlove — APPROVED** [review]

**Review by malangfox — APPROVED** [review]
LGTM

File: src/core/annotation/AnnotationManager.ts [hunk 1]
```diff
@@ -151,19 +151,37 @@ class AnnotationManager {
    * Parse an array of annotation data
    * @param {object[]} data An array of annotation data
    */
-  public parse(data: Array<Record<string, any>>): Annotation[] {
+  public parse(data: {
+    baseFov: number;
+    baseDistance: number;
+    aspect: number;
+    items: Array<{
+      meshIndex: number;
```
Comment (daybrush) [inline src/core/annotation/AnnotationManager.ts:159]: Although the type of meshIndex is a required number, it seems okay to specify optional and null types with 171 line

**Review by daybrush — APPROVED** [review]
LGTM

File: src/core/annotation/AnnotationManager.ts (hunk 1 above)
Comment (WoodNeck) [reply src/core/annotation/AnnotationManager.ts:159]: Good point 👍

**Pull request merged**